src/runtime.js

```diff
@@ -359,6 +370,27 @@ let tornado = {
   },
 
   /**
+   * Build a pending body within a div with class "pending" (we have to wrap in a div so we can
+   * easily replace the entire thing when the promise resolves). If no pending body exists, then
+   * return false.
+   * @param {Node} placeholderNode The node where the pending body will be inserted
+   * @param {[Function]} body The pending body function, if it exists
+   * @param {Object} context The current Tornado context, to be used in building the pending body
+   * @return {HTMLElement|False} Return the containing div, or false
+   */
+  insertPendingBody(placeholderNode, body, context) {
+    if (body) {
+      let div = document.createElement('div');
+      div.setAttribute('class', 'pending');
```

Comment:
We could change the class to `tornado-pending`, or add the ability to customize that class name, but I would wait for a bug report before we do either.